src/scenes/Event/components/EventTxHistory/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
```

Comment:
Why do you have this file?

Reply:
I pulled from master this Monday and added a few lines into it.

If this is meant to be deleted, I can delete and push again.

Reply:
it might be because my editor made some mistakes, I'm changing over to VScode